CHANGELOG.md

```diff
@@ -2,15 +2,17 @@
 
 ## Unreleased
 
+### Fixes
+
+- Android: Fix for SIGSEGV reported by Sentry when a NullReferenceException is thrown in Release mode ([#3694](https://github.com/getsentry/sentry-dotnet/pull/3694))
+  - To turn it on, set: `options.HandlerStrategy = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_CHAIN_AT_START;`
+- Events from NDK on Android will report sdk.name `sentry.native.android.dotnet` ([#3682](https://github.com/getsentry/sentry-dotnet/pull/3682))
```

Comment:
Isn't this a comment from another PR?

Reply:
Not an open one at least. 
this is on [main](https://github.com/getsentry/sentry-dotnet/blob/main/CHANGELOG.md#L12) for sure

Reply:
Ah I see, it's just been moved from below the `###Features` in this branch. I was wondering why it was showing as being added in this PR.

Reply:
yeah I see

do you think I should change that?

Reply:
Historically we've put the features before the Fixes... just a question of what we want to draw people's attention to first. Not a big deal.